### ISSUE TITLE
Drag-Dropping of Tables from LibreOffice Calc / Excel

### ISSUE THREAD

**Issue by geolta** [texstudio-org/texstudio#2672]
Creating tables in LaTeX is always a bit annoying if it is based on table already existing in a spreadsheet application. Similar to drag-dropping images I would suggest to add drag-dropping and copy-pasting tables to TXS. 

I could implement this.

**Comment by sunderme** [comment]
there is some functionality with the tabular wizard, but it is not very extensive

**Comment by geolta** [comment]
Yeah, I'm more thinking about just converting the table to LaTeX format or in other words separate cells by `&` and newlines by `\\` without starting the wizard. Maybe also putting this into a tabular environment by default.
